main.py

```diff
@@ -100,37 +96,46 @@ def draw(self):
 class Astroid(Sprite):
     
     def __init__(self):
-        self.size = (random.randint(25, 50), random.randint(25, 50))
-        self.speed1 = random.randint(-5, 5)
-        self.speed2 = random.randint(-5, 5)
         self.x = random.randint(0, MAX_SIZE_X)
-        self.y = 0
+        self.y = random.randint(0, MAX_SIZE_Y)
```

Comment:
This will potentially start the asteroid inside the ship, so we should keep it starting at y=0 for now.